components/OAuth2.0/OauthHttpLocalServer.c

```diff
@@ -1,19 +1,18 @@
-#include "SpotifyHttpLocalServer.h"
-#include "SpotifyInterface.h"
-#include "SpotifyTypedef.h"
+#include "OauthHttpLocalServer.h"
+#include "authorization.h"
 
-static const char *TAG = "SpotifyTask";
+static const char *TAG = "OauthTask";
 // ******************************
 
-QueueHandle_t SendCodeFromHttpToSpotifyTask = NULL;
+QueueHandle_t SendCodeFromHttpToTask = NULL;
 
 /**
  * @brief This function searches for specific patterns ('code' and 'state') within a character array and returns a boolean value indicating if either pattern was Found.
- * @param[in] Response The character array to search within, and Response is response from first stage from spotify athurisiation
+ * @param[in] Response The character array to search within, and Response is response from first stage from Service athurisiation
  * @param[in] SizeRes The size of the character array.
  * @return Returns true if either the 'code' or 'state' pattern was Found, and false otherwise.
  */
-bool Spotify_FindCode(char *Response, uint16_t SizeRes)
+bool FindCode(char *Response, uint16_t SizeRes)
 {
     char *codeString = {"code"};
```

Comment:
is the code always the same for different Oauth providers? if not, it should be reconfigurable value

Reply:
It just searches for the code and is always the same way